service-base-utils-parent-os/whBldToolsInstall.bash

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash -u
+#  ************************************** #
+#  (C) Copyright IBM Corp. 2001, 2020     #
+#  SPDX-License-Identifier: Apache-2.0    #
+#  ************************************** #
+
+#:Functions
+BATCH_NAME=whBldToolsInstall.bash
+BATCH_VER=2017.06.19.0
+BATCH_OWNER=SHachem
```

Comment:
Not sure what this does or if this is a real person?

Reply:
Confirmed that this file and the other whBldToolsXXX files can be removed. I will do that.